src/node/consumer/src/lib/helpers/boolean_helpers.ts

```diff
@@ -0,0 +1,20 @@
+export const BooleanHelpers = {
+    parseBool: (value: string | undefined, defaultValue: boolean): boolean => {
+        if (value === undefined) {
+            return defaultValue;
+        }
+
+        switch (value.trim().toLowerCase()) {
+            case 'true':
```

Comment:
The if statement can be added to the switch like this:

```suggestion
        switch (value?.trim()?.toLowerCase()) {
            case undefined:
                return defaultValue;
            case 'true':
```

Looks a bit nicer imo.

Reply:
Agreed - also rather than falling back to default value when something invalid is entered (in case of typo's in env vars), i've added an error message stating the boolean was rejected and outputting the possible values